File: userService/userService/angular/src/app/shared/dto/EventoFeignReserva.ts
export interface EventoFeignReserva {
    idEvento: number;
    nombre: string;
    descripcion: string;
    tematica: string;
    fechaInicio: string; // LocalDateTime → string ISO
    fechaFin: string; // LocalDateTime → string ISO
    aforo: number;
    imagen: string;
}

File: eventService/eventService/src/main/java/com/koroFoods/eventService/dtos/EventoFeignReserva.java
package com.koroFoods.eventService.dtos;

import java.time.LocalDateTime;

import lombok.Data;

@Data
public class EventoFeignReserva {

    private Integer idEvento;
    private String nombre;
    private String descripcion;
    private String tematica;
    private LocalDateTime fechaInicio;
    private LocalDateTime fechaFin;
    private Integer aforo;
    private String imagen;
}
